python/tskit/drawing.py

```diff
@@ -280,8 +360,10 @@ def __init__(
         node_label_attrs=None,
         mutation_attrs=None,
         mutation_label_attrs=None,
+        svg_id=None,
```

Comment:
I thought we were moving away from using SVG IDs now? It doesn't seem to be used anyway, so let's remove it unless you have some specific plan.

Reply:
While I'm at it, can I make most of the params (perhaps after `size`) keyword only? Another param that might be useful is `styles` to embed your own stylesheet info into the top of the svg, and take advantage of these nice new features without having to hand-edit the svg or embed it in an html file. Do you want me to add this to the current PR @jeromekelleher , or submit it as another?

Reply:
Let's do any further svg extensions  as a follow up. Don't worry about the interfaces within the drawing.py module, these aren't exposed anyway. Making things kw only is probably a good idea in draw_svg etc.

Reply:
@benjeffery - re styles, I was imagining being able to pass e.g. `.nodes .n10 {stroke: red}` to the draw_svg function somehow, which would create

`<svg... ><style>.nodes .n10 {stroke: red}</style>...</svg>`

I don't think that can be done as an attribute on the svg tag, can it?

Reply:
Nope, you'd need another param. If a use case here is stand-alone SVG then this makes sense - if it is just for HTML then the caller can add css themselves.

Reply:
Yes, I do think that some people will want to create stand-alone SVGs. If we add a "styles" parameter, this would save them the necessity of hand-editing the SVG, which I think is worth it, especially if we are advising people that this is a simple way to e.g. hide certain labels etc.